src/librustc_mir/borrow_check/nll/region_infer/mod.rs

```diff
@@ -404,22 +404,6 @@ impl<'tcx> RegionInferenceContext<'tcx> {
         upvars: &[Upvar],
         mir_def_id: DefId,
         errors_buffer: &mut Vec<Diagnostic>,
-    ) -> Option<ClosureRegionRequirements<'tcx>> {
-        common::time_ext(
```

Comment:
Probably should just keep this and use `-Z time` instead? I don't think anyone will miss it though.

Reply:
On the contrary, why should it even be there?

Reply:
Giving it some more thought, I'm going to go a step further and remove -Ztime. (Or perhaps leave it as a synonym for -Ztime-passes.) It's silly to have two timing flags, where one is useless (the current situation). It's also silly to have two timing flags that give almost-identical output (the situation with this patch).

Reply:
It makes sense to have a high level and a more detailed timing pass to me, but maybe the detailed stuff should just be left to `-Z self-profile`.

Reply:
Eh, I'll leave the `time-passes`/`time` split for now even though I don't like it. But I strongly believe that removing the `solve_nll_region_constraints` entries is the right decision; they are zero-value spam.